db2/wavefront_db2_metrics.py

```diff
@@ -0,0 +1,521 @@
+# DB2 integration which collect DB2 performance data
+# and delivers to Wavefront Server, either via proxy or via direct ingestion
+# you can have it run via telegraf's exec plugin or run it using other scheduler
+import argparse
+import ibm_db
+import sys
+import time
+import datetime
+import calendar
+import socket
+import os
+
+# global settings
+interval = 0
+
+# format can be either [wavefront|influx]
+out_format = "influx"
+debug = False
+
+# output type
+# [telegraf|wf_proxy|wf_direct]
+# telegraf: send the data to std output for telegraf's exec plugin to receive
+# wf_proxy: send the data to wavefront proxy (requires host and port)
+# wf_direct (not supported yet): send the data directly to wavefront (requires url and app token)
+out_type = "telegraf"
+
+# proxy setting which will be used when
+# wf_proxy settings
+proxy_host = "localhost"
+proxy_port = 2878
+
+# direct data ingestion settings
+# wf_direct
+direct_url = "https://longboard.wavefront.com"
```

Comment:
longboard is a production cluster, so lets not give that as an example.
we can just say https://<INSTANCE>.wavefront.com

Reply:
OK